src/point_one/fusion_engine/messages/solution.h

```diff
@@ -51,7 +54,8 @@ struct P1_ALIGNAS(4) PoseMessage : public MessagePayload {
   /** The type of this position solution. */
   SolutionType solution_type;
 
-  uint8_t reserved = 0;
+  /** Signals device dynamic status (stationary or nonstationary). */
+  uint8_t dynamic_status = NONSTATIONARY;
```

Comment:
We want this to be a more general purpose flags bitmask, so I'd suggest calling it `flags` and documenting it similar to flags variables in other messages like `WheelSpeedInput` for example. Something like "A bitmask of flags indicating the status of this solution." It doesn't have to be limited to stationary or not, we might also define bits for other things in the future.

For the constants, name it `FLAG_STATIONARY = 0x01` (it's an 8b mask, so don't need all the additional 0s) and get rid of `NONSTATIONARY` since that's not defining a bit, it's just the lack of one particular bit.

The `flags` variable should default to 0x0.